lib/runner/extensions/request.command.js

```diff
@@ -190,9 +205,10 @@ module.exports = {
                         context.response = res;
 
                         // run the post request helpers, which need to use the response, assigned above
-                        async.applyEachSeries(posthelpers, context, self, function (/* err */) {
+                        async.applyEachSeries(posthelpers, context, self, function (err, options) {
+                            if (err) { /** */ }
```

Comment:
handle error here? question is, what nature of error will bubble up here and then are they supposed to stop run?

Reply:
Handled error and enabled test cases for asserting the error.